test/controllers/nonsipp/memberdetails/SchemeMemberDetailsAnswersControllerSpec.scala

```diff
@@ -223,10 +223,10 @@ class SchemeMemberDetailsAnswersControllerSpec extends ControllerBaseSpec {
             pureUserAnswers = userAnswersWithNino.unsafeSet(MemberStatus(srn, refineMV(1)), MemberState.New).some,
             expectations = (ua: UserAnswers) =>
               List(
-                ua.get(MemberStatus(srn, refineMV(1))).exists(_.changed),
+                ua.get(MemberStatus(srn, refineMV(1))).exists(_._new),
                 ua.get(SafeToHardDelete(srn, refineMV(1))).isEmpty
               )
-          ).withName(s"Set Member status to Changed when a member detail has changed")
+          ).withName(s"Set Member status to New when a member detail has changed")
```

Comment:
the name of this test doesn't make sense, also this test should have stayed the same, if it failed it means something is missing (completed page is missing from test useranswers / wrong mode, etc)

If anything a new test case should have been written for the additional logic

Reply:
Hi @westwater, could you please check now, thanks

Reply:
@narendrapaduchuri I think the second part of @westwater's comment is not addressed: 
"If anything a new test case should have been written for the additional logic", in my understanding that is to add a test where `call = onSubmit(CheckMode)` and MemberDetailsCompletedPage is empty in user answers. To check the additional logic you have added.